extensions/amp-form/0.1/amp-form-textarea.js

```diff
@@ -144,6 +132,30 @@ export class AmpFormTextarea {
   }
 }
 
+/**
+ * For now, warn if textareas with initial overflow are present, and
+ * prevent them from becoming autoexpand textareas.
+ * @param {!IArrayLike<!Element>} textareas
+ * @return {!Promise}
+ */
+export function handleInitialOverflowElements(textareas) {
+  const promises = [];
+  iterateCursor(textareas, element => {
```

Comment:
Would consider:

```javascript
const promises = Array.from(textareas).map(element) => {
  return getHasOverflow(element).then(...);
});
```

The cost of creating an intermediate array here is so small and using a map reads more cleanly.

Reply:
I believe `Array.from` is not allowed yet in amphtml to avoid the polyfill bytes

Reply:
I keep getting tripped on that. You can use `toArray`: https://github.com/ampproject/amphtml/blob/fbe3402006c8f17d37f818f141f0ed826eee7eed/src/types.js#L44-L46